packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.10.0
+
+* **Breaking Change** Bumps default camera_web package version to 0.3.0.
+* **Breaking Change** Updates Android camera access permission error codes to be consistent with other platforms.
```

Comment:
I think it would need to provide the steps to migrate from the previous version to the new version

Reply:
Can I ask for clarification here? Do you mean to note the new error code used?

Reply:
I think what @blasten meant (let me know if i misunderstood) is to add some migration instruction here, something like: 

"If you use handle the legacy `cameraPermission` exception code, please update it to use the new exception codes as described in the README. "

Reply:
@blasten I updated it, but if you were referencing something else, just let me know!